src/actions/userActions.js

```diff
@@ -59,3 +64,33 @@ export const loginUser = payload => async dispatch => {
       );
     });
 };
+
+export const googleLoginUser = (serviceProvider, userData) => (dispatch) => {
+  dispatch(socialLoginInitiated(true));
+  return axiosInstance.post(serviceProvider, userData)
+    .then((res) => {
+      localStorage.setItem('auth_token', res.data.user.access_token);
+      dispatch(socialLoginSuccess(true));
+      toast.success('Signup successful', { autoClose: 3500, hideProgressBar: true });
+    })
+    .catch(() => {
+      toast.error('We can not sign you in if you registered with another social account', { autoClose: 3500, hideProgressBar: true }, {
+        position: toast.POSITION.TOP_CENTER,
+      });
+    });
+};
+
+export const facebookLoginUser = (serviceProvider, userData) => (dispatch) => {
+  dispatch(socialLoginInitiated(true));
+  return axiosInstance.post(serviceProvider, userData)
+    .then((res) => {
+      localStorage.setItem('auth_token', res.data.user.access_token);
+      dispatch(socialLoginSuccess(true));
+      toast.success('Signup successful', { autoClose: 3500, hideProgressBar: true });
+    })
+    .catch(() => {
+      toast.error('We can not sign  you in if you registered with another social account', { autoClose: 3500, hideProgressBar: true }, {
```

Comment:
Please remove the extra space between `sign  you`
Plus the error message is a bit long and vague. With a bit more clarity on what causes this error, I am certain we can find a better error message.